module/actor/sheet/actor-sheet.js

```diff
@@ -29,9 +29,58 @@ export default class PBActorSheet extends (foundry.appv1?.sheets?.ActorSheet ??
   constructEffectLists(sheetData) {
     const effects = {};
 
-    effects.temporary = sheetData.actor.effects.filter((i) => i.isTemporary && !i.disabled && !i.isCondition);
-    effects.disabled = sheetData.actor.effects.filter((i) => i.disabled && !i.isCondition);
-    effects.passive = sheetData.actor.effects.filter((i) => !i.isTemporary && !i.disabled && !i.isCondition);
+    // Helper function to add modifier display to effects
+    const addModifierDisplay = (effect) => {
+      const modifiers = [];
+      effect.changes.forEach(change => {
+        if (change.key.includes('combat.')) {
+          const modifier = change.key.replace('system.attributes.combat.', '');
+          const value = change.value;
+          let displayName = '';
+          
+          switch (modifier) {
+            case 'attackModifier': displayName = 'Attack'; break;
+            case 'defenseModifier': displayName = 'Defense'; break;
+            case 'initiativeModifier': displayName = 'Initiative'; break;
+            case 'damageModifier': displayName = 'Damage'; break;
+            case 'armorTierModifier': displayName = 'Armor Tier'; break;
+            case 'speedModifier': displayName = 'Speed'; break;
+            case 'luckDieModifier': displayName = 'Luck Die'; break;
+          }
+          
+          if (displayName && value != 0) {
+            modifiers.push(`${value > 0 ? '+' : ''}${value} ${displayName}`);
+          }
+        } else if (change.key.includes('abilities.') || change.key.includes('attributes.')) {
+          // Handle other attribute modifiers
+          const parts = change.key.split('.');
+          const attrName = parts[parts.length - 2] || parts[parts.length - 1];
+          const value = change.value;
+          
+          if (value != 0) {
+            let displayName = attrName.charAt(0).toUpperCase() + attrName.slice(1);
+            if (parts.includes('max')) displayName += ' Max';
+            else if (parts.includes('value')) displayName += ' Current';
```

Comment:
What are your thoughts on `Current` vs `Value`? I'm not sure which is clearer in the context of effects.

Reply:
I waffled in my language a few times throughout so far already, will need to standardize the verbage at some point.

I think current is helpful as it shows that this could change

Reply:
Could take a look at the 5e system and see what they have. their effects system is incredibly robust